_data/codebase/v2_3/open-source/bin-magento.yml

```
@@ -5145,14 +5149,22 @@ commands:
         is_value_required: true
         is_multiple: false
         description: The preferred area (global, adminhtml, etc...) default is global.
-        default: 
+        default:
+      single-thread:
+        name: "--single-thread"
+        shortcut: ''
+        accept_value: false
+        is_value_required: false
+        is_multiple: false
+        description:
+        default: false
       pid-file-path:
         name: "--pid-file-path"
         shortcut: ''
         accept_value: true
         is_value_required: true
         is_multiple: false
-        description: The file path for saving PID
+        description: The file path for saving PID (This option is deprecated, use --single-thread instead)
```

```suggestion
        description: This option is deprecated. Use  the ```--single-thread`` option instead.
```